composer.json

```diff
@@ -32,16 +32,16 @@
         "composer-plugin-api": "^1.1 || ^2.0"
     },
     "require-dev": {
-        "composer/composer": "^1.9",
-        "friendsofphp/php-cs-fixer": "^2.16",
+        "composer/composer": "^1.9 || ^2.0",
+        "friendsofphp/php-cs-fixer": "^2.18, <2.19",
```

Comment:
cs fixer 2.19 shows a lot of deprecation messages and it will be updated in the separate PR